english/vote-the-president/Hillary.md

```diff
@@ -0,0 +1 @@
+Congratulations. You made another clinton president again. Get ready to see some great extra marital affairs that will give a lots of money to media and spicy news for the Americans.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.